go.mod

```diff
@@ -1,214 +1,292 @@
 module github.com/morningconsult/docker-credential-vault-login
 
-go 1.17
+go 1.20
 
 require (
-	github.com/docker/docker-credential-helpers v0.7.0
-	github.com/google/go-cmp v0.5.7
-	github.com/hashicorp/go-hclog v1.4.0
+	github.com/docker/docker-credential-helpers v0.8.0
+	github.com/google/go-cmp v0.5.9
+	github.com/hashicorp/go-hclog v1.5.0
 	github.com/hashicorp/go-uuid v1.0.3
-	github.com/hashicorp/vault v1.11.4
-	github.com/hashicorp/vault/api v1.7.2
-	github.com/hashicorp/vault/sdk v0.5.3-0.20220826205147-7caf353e8124
+	github.com/hashicorp/hcl v1.0.1-vault-5
+	github.com/hashicorp/vault v1.15.0
+	github.com/hashicorp/vault/api v1.10.0
+	github.com/hashicorp/vault/sdk v0.10.1-0.20230922151601-0fa36a36ae1b
```

Comment:
Just curious, how'd you find the relevant commit?

Reply:
I actually just picked the latest commit that changed files in the `github.com/hashicorp/vault/sdk` package 
https://github.com/hashicorp/vault/tree/main/sdk